eZ/Publish/API/Repository/Values/URL/SearchResult.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace eZ\Publish\API\Repository\Values\URL;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class SearchResult extends ValueObject implements \IteratorAggregate
+{
+    /**
+     * @var int
+     */
+    protected $count;
```

Comment:
`public $totalCount;`

Like in `Content\Search\SearchResult`, prop below can also be public, no need to force magic call on this

Reply:
Done in 37ad13409249375cd115327f94fb34f9d2d2b5e3.